pycbc/io/live.py

```diff
@@ -251,6 +257,29 @@ def __init__(self, ifos, coinc_results, **kwargs):
             psds_lal[ifo] = fseries
         make_psd_xmldoc(psds_lal, outdoc)
 
+        # source probabilities estimation
```

Comment:
I am a bit skeptical of having this block of code in the `io` module. It seems estimating distance, dealing with redshift etc belong somewhere else. I suggest doing this calculation in the `mchirp_area` module and then passing `probabilities` as a kwarg when instantiating the `SingleCoincForGraceDB` object.